ramls/oaipmh/oaipmhupdatedinstanceids.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response schema for oai-pmh updated instances ids view",
+  "type": "object",
+  "properties": {
+    "instanceId": {
+      "description": "OAI-PMH updated instances ids",
+      "type": "string"
+    },
+    "instanceUpdatedDate": {
+      "description": "The last updated date or deleted date of an instance or it's items and holdings",
+      "type": "string",
+      "format": "date-time"
+    },
+    "suppressDiscovery": {
```

Comment:
Similar to one of @bohdan-suprun earlier comments, could this be `suppressFromDiscovery` instead, or is it like this for consistency with something else?

Reply:
As I mentioned above this name came from data view and for the sake of minimizing any changes this name was retained. If it looks confusing I think we can fix this. So the last word is for you, let me know about your final decision and I will apply it.

Reply:
I think if it is named this way for consistency that is ok. I would have preferred the name to read better, that opportunity is likely now lost to us for the time being.

Reply:
I will fix it. Thank you for noticing.